ci/travis/test_script.sh

```diff
@@ -1,5 +1,7 @@
 #! /bin/bash
 
+set -e
```

Comment:
This is causing the doc build to fail on line 39 below, which can just be deleted (I can't work out how to open a PR on your branch, but https://travis-ci.org/dstansby/matplotlib/builds/221288308 is my travis build)